### DISCUSSION TITLE
Adding unlimited optional digits

### DISCUSSION THREAD

**Discussion by vzelenko** [uNmAnNeR/imaskjs#706] [General]
Suppose I want to create a mask for phone numbers that looks something like this:

(000) 000-0000`[xEXT]`

where `[xEXT]` is optional, and EXT can be any number of digits.

How can I specify `[xEXT]`?

**Comment by uNmAnNeR** [comment]
you can try
1) using pattern + blocks with regexp inside
2) using dynamic mask
3) think that phones are limited in length and apply strict mask

**Reply by vzelenko** [reply]
Thank you.  I used pattern, and limited phone number extension to a maximum of 6 digits.
